libparsec/crates/types/src/token.rs

```diff
@@ -0,0 +1,152 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+/// The size of the generated tokens.
+const TOKEN_SIZE: usize = 16;
+/// The minimal size of the generated tokens.
+/// This is used to ensure that the tokens are not too small during deserialization.
+const TOKEN_MINIMAL_SIZE: usize = 16;
+
+#[derive(thiserror::Error, Debug)]
+pub enum TokenDecodeError {
+    #[error("The token is not a valid hex string")]
+    InvalidHex,
+    #[error(
```

Comment:
I we remove `TOKEN_MINIMAL_SIZE`, this error should be renamed `InvalidSize`

Reply:
The discussion will continue in https://github.com/Scille/parsec-cloud/pull/6984#discussion_r1547370876